Model/BalanceBuyer.php

```diff
@@ -85,6 +86,41 @@ public function __construct(
         $this->balancepayConfig = $balancepayConfig;
     }
 
+    /**
+     * CreateBuyer
+     *
+     * @param array $params
+     * @return false|void
+     * @throws LocalizedException
+     * @throws NoSuchEntityException
+     */
+    public function createBuyer($params = [])
+    {
+        if (!empty($params['email'])) {
+            try {
+                $response = $this->requestFactory
+                    ->create(RequestFactory::BUYER_REQUEST_METHOD)
+                    ->setRequestMethod('buyers')
+                    ->setTopic('buyers')
+                    ->setParams($params)
+                    ->process();
+            } catch (Exception $e) {
+                $this->balancepayConfig->log('Create buyer [Exception: ' .
```

Comment:
- log level should be warn.
- Why not throw the exception instead of returning false?

Reply:
Removed try catch from here.

Reply:
why? no need to log?

Reply:
Exception will be caught in the parent function.